### DISCUSSION TITLE
A way to sync visual clips with audio clips in vegas... without a MIDI file

### DISCUSSION THREAD

**Discussion by Youmg44** [otomad/OtomadHelper#48] [Ideas]
![2023-07-20_01h44_04](https://github.com/otomad/OtomadHelper/assets/80075586/be146536-eb13-4d44-b67a-c2e66966e592)

Is it possible to have otoMAD helper fill out the video clip above the audio clips in the same .veg file **without requiring a MIDI?**

**Comment by otomad** [comment]
The [lastest version](https://github.com/otomad/OtomadHelper/releases/latest) add a new tool, you can export a MIDI file from your audio track events, then you can reimport the MIDI file to Otomad Helper to generate visuals.
